.dotnet/api/OpenAI.netstandard2.0.cs

```diff
@@ -1232,14 +1232,14 @@ public class ChatClient {
     }
     public class ChatCompletion : IJsonModel<ChatCompletion>, IPersistableModel<ChatCompletion> {
         public IReadOnlyList<ChatMessageContentPart> Content { get; }
-        public IReadOnlyList<ChatTokenLogProbabilityInfo> ContentTokenLogProbabilities { get; }
+        public IReadOnlyList<ChatTokenLogProbabilityDetails> ContentTokenLogProbabilities { get; }
         public DateTimeOffset CreatedAt { get; }
         public ChatFinishReason FinishReason { get; }
         public ChatFunctionCall FunctionCall { get; }
         public string Id { get; }
         public string Model { get; }
         public string Refusal { get; }
-        public IReadOnlyList<ChatTokenLogProbabilityInfo> RefusalTokenLogProbabilities { get; }
+        public IReadOnlyList<ChatTokenLogProbabilityDetails> RefusalTokenLogProbabilities { get; }
```

Comment:
I wonder if `ChatTokenLogProbabilityDetails` and `ChatTokenTopLogProbabilityDetails` should be structs.